packages/foreman/foreman-installer/foreman-installer.spec

```diff
@@ -1,17 +1,15 @@
-# set and uncomment all three to set alpha tag
-#global alphatag RC1
-#global dotalphatag .%{alphatag}
-#global dashalphatag -%{alphatag}
+%global prerelease .develop
+%global release 2
```

Comment:
We're still on 1 here

Reply:
Since I am making a change to the spec, I feel like the release should get bumped to reflect that.

Reply:
Technically the old release was on 0 so 1 is still an increment :)